v2/internal/controllers/crd_apimanagement_20220801_test.go

```diff
@@ -111,6 +113,24 @@ func Test_ApiManagement_20220801_CRUD(t *testing.T) {
 				APIM_Api_CRUD(tc, &service)
 			},
 		},
+		testcommon.Subtest{
+			Name: "APIM Authorization Provider CRUD",
+			Test: func(tc *testcommon.KubePerTestContext) {
+				APIM_AuthorizationProvider_CRUD(tc, &service)
+			},
+		},
+		testcommon.Subtest{
+			Name: "APIM Authorization CRUD",
+			Test: func(tc *testcommon.KubePerTestContext) {
+				APIM_Authorization_CRUD(tc, &service)
```

Comment:
Better to rename this to something like `APIM_AuthorizationProviders_Authorizations_CRUD`, and same for the other tests as well